Problem with ESLint parser when running Typescipt

**Description:**
Experiencing Problem with using either the `tsNode` or `tsReact` plugin. The following Error keep persisting

```bash
/Users/tamv/Versett/@versett/versett-boilerplate/packages/api/src/server.ts
   6:51  error    Parse errors in imported module './middleware': Cannot find module 'typescript-eslint-parser' (undefined:undefined)    import/named
   7:20  error    Parse errors in imported module './routes': Cannot find module 'typescript-eslint-parser' (undefined:undefined)        import/no-named-as-default
   7:20  warning  Parse errors in imported module './routes': Cannot find module 'typescript-eslint-parser' (undefined:undefined)        import/no-named-as-default-member
   7:20  error    Parse errors in imported module './routes': Cannot find module 'typescript-eslint-parser' (undefined:undefined)        import/default
   8:20  error    Parse errors in imported module './utils/getEnv': Cannot find module 'typescript-eslint-parser' (undefined:undefined)  import/default
   8:20  error    Parse errors in imported module './utils/getEnv': Cannot find module 'typescript-eslint-parser' (undefined:undefined)  import/no-named-as-default
   8:20  warning  Parse errors in imported module './utils/getEnv': Cannot find module 'typescript-eslint-parser' (undefined:undefined)  import/no-named-as-default-member
   9:21  error    Parse errors in imported module './graphql': Cannot find module 'typescript-eslint-parser' (undefined:undefined)       import/default
   9:21  error    Parse errors in imported module './graphql': Cannot find module 'typescript-eslint-parser' (undefined:undefined)       import/no-named-as-default
   9:21  warning  Parse errors in imported module './graphql': Cannot find module 'typescript-eslint-parser' (undefined:undefined)       import/no-named-as-default-member
  11:16  error    Missing return type on function         
```
**Steps to Reproduce.**
1. Pull latest from `https://github.com/versett/versett-boilerplate`
2. navigate to `/packages/api (for testing the tsNode)`
3. currently run yarn lint and it will pass all linting. (Currently use eslint 4.19.0)
4. update to eslint 5.6.0 (same version as Create-React-App)
5. delete all eslint package and follow the steps provide by the readme.md of this project.
6. delete `node_modules` and run `yarn` to make usre everything is fresh.
7. udpate `.eslintrc` to match the read me with the plugin set to tsNode
8. then run yarn eslint eslint --ext .ts src to test just the `src` folder.

**Requirements / AC:**
If you revert all of this back to the original code in master and run yarn lint it will not cause any proble.

--- a/configurations/js-node.js
+++ b/configurations/js-node.js
@@ -44,6 +44,5 @@ module.exports = {
         extensions: [".js"]
       }
     }
-  },
-  rules: {}
+  }
 };

--- a/configurations/js-react.js
+++ b/configurations/js-react.js
@@ -51,6 +51,5 @@ module.exports = {
     react: {
       version: "detect"
     }
-  },
-  rules: {}
+  }
 };

--- a/configurations/ts-node.js
+++ b/configurations/ts-node.js
@@ -10,14 +10,14 @@ module.exports = {
   extends: [
     "plugin:import/errors",
     "plugin:import/warnings",
-    "plugin:import/typescript",
     require.resolve("../configurations/eslint-all"),
     require.resolve("../rules/jest/on"),
     require.resolve("../rules/ts/on")
   ],
   env: {
     jest: true,
-    node: true
+    node: true,
+    es6: true
   },
   globals: {
     global: true,
@@ -44,6 +44,5 @@ module.exports = {
         extensions: [".ts"]
       }
     }
-  },
-  rules: {}
+  }
 };

--- a/configurations/ts-react.js
+++ b/configurations/ts-react.js
@@ -11,14 +11,14 @@ module.exports = {
     "plugin:jsx-a11y/recommended",
     "plugin:import/errors",
     "plugin:import/warnings",
-    "plugin:import/typescript",
     require.resolve("../configurations/eslint-all"),
     require.resolve("../rules/react/on"),
     require.resolve("../rules/jest/on"),
     require.resolve("../rules/ts/on")
   ],
   env: {
-    jest: true
+    jest: true,
+    es6: true
   },
   globals: {
     global: true,
@@ -51,6 +51,5 @@ module.exports = {
     react: {
       version: "detect"
     }
-  },
-  rules: {}
+  }
 };

